### ISSUE TITLE
Prevent pip while using quick download 

### ISSUE THREAD

**Issue by venkat2305** [JunkFood02/Seal#506]
### Checklist

- [ ] This feature I'm requesting is already implemented in yt-dlp.
- [X] This feature is merely a UI/UX update.
- [X] This feature is suitable for primary users with little knowledge about yt-dlp.
- [X] This feature is available for most websites, not only the video platform I use.
- [X] This feature is suitable for a large variety of videos.
- [X] This feature is not going to conflict with many of the existing options.

### Is your feature request related to a problem? Please describe.

I turned off formation selection and configure before download options. While the video is playing if use quick download or the download button in revanced, the app is minimising to home and opening pip. There's no issue if we do that while the video is paused 

### Describe the solution you'd like

Feature [tldr]: Prevent pip and stop the YouTube app from minimising

**Comment by venkat2305** [comment]
https://user-images.githubusercontent.com/55329452/212093159-4531eba2-c848-4ec1-a265-9d3247990741.mp4

**Comment by Gittore54** [comment]
> https://user-images.githubusercontent.com/55329452/212093159-4531eba2-c848-4ec1-a265-9d3247990741.mp4

![20230116_053436.jpg](https://user-images.githubusercontent.com/113577927/212600290-f9ce06ad-7c18-40ea-bcad-374a2b364f78.jpg)

If I can ask, what is this?

**Comment by venkat2305** [comment]
It can stream anime. It scrapes from various websites.

**Comment by Gittore54** [comment]
> It can stream anime. It scrapes from various websites. 

Thanks, It's on github?

**Comment by venkat2305** [comment]
Yes

**Comment by Gittore54** [comment]
> Yes

Name?

**Comment by Marco-9456** [comment]
@Gittore54 Use Saikou it's also on GitHub

**Comment by Gittore54** [comment]
> @Gittore54 Use Saikou it's also on GitHub

Yeah know it, Aniyomi too, but didnt recognize this, probably I have already starred but never downloaded or installed

**Comment by fedonr** [comment]
+1 for this feature

**Comment by Francesco146** [comment]
Still happening on v1.8.1 and pre-release version.

**Comment by Francesco146** [comment]
this might be a android related problem, not sure what Seal can do.

**Comment by venkat2305** [comment]
> this might be a android related problem, not sure what Seal can do.

this problem is specific to seal, since new pipe and powertube don't have this issue.

**Comment by Marco-9456** [comment]
> 

Seal is programmed using completely different and new libraries that the apps you mentioned don't use, so it's normal that JunkFood02 can't fix them because these are probably issues with Jetpack Compose that haven't been fixed from Google's side.

In other words, these are external issues that probably can't be fixed in a short period.

Though JunkFood02 is a very good programmer he might surprise us with a workaround.
